src/main/java/frc/lib/util/swerveUtil/COTSNeoSwerveConstants.java

```diff
@@ -0,0 +1,204 @@
+
+/* Contains values and required settings for common COTS swerve modules. */
+public class COTSNeoSwerveConstants {
+    public final double wheelDiameter;
+    public final double wheelCircumference;
+    public final double angleGearRatio;
+    public final double driveGearRatio;
+    public final double angleKP;
+    public final double angleKI;
+    public final double angleKD;
+    public final double angleKF;
+    public final boolean driveMotorInvert;
+    public final boolean angleMotorInvert;
+    public final boolean canCoderInvert;
+
+    public COTSNeoSwerveConstants(double wheelDiameter, double angleGearRatio, double driveGearRatio, double angleKP, double angleKI, double angleKD, double angleKF, boolean driveMotorInvert, boolean angleMotorInvert, boolean canCoderInvert){
+        this.wheelDiameter = wheelDiameter;
+        this.wheelCircumference = wheelDiameter * Math.PI;
+        this.angleGearRatio = angleGearRatio;
+        this.driveGearRatio = driveGearRatio;
+        this.angleKP = angleKP;
+        this.angleKI = angleKI;
+        this.angleKD = angleKD;
+        this.angleKF = angleKF;
+        this.driveMotorInvert = driveMotorInvert;
+        this.angleMotorInvert = angleMotorInvert;
+        this.canCoderInvert = canCoderInvert;
+    }
+    
+    /** Swerve Drive Specialties - MK3 Module*/
+    public static COTSNeoSwerveConstants SDSMK3(double driveGearRatio){
+        double wheelDiameter = Units.inchesToMeters(4.0);
+ 
+        /** 12.8 : 1 */
+        double angleGearRatio = (12.8 / 1.0);
+ 
+        double angleKP = 0.2;
+        double angleKI = 0.0;
+        double angleKD = 0.0;
+        double angleKF = 0.0;
+ 
+        boolean driveMotorInvert = false;
+        boolean angleMotorInvert = false;
+        boolean canCoderInvert = false;
+        return new COTSNeoSwerveConstants(wheelDiameter, angleGearRatio, driveGearRatio, angleKP, angleKI, angleKD, angleKF, driveMotorInvert, angleMotorInvert, canCoderInvert);
+    }
+
+    /** Swerve Drive Specialties - MK4 Module*/
+    public static COTSNeoSwerveConstants SDSMK4(double driveGearRatio){
+        double wheelDiameter = Units.inchesToMeters(4.0);
+ 
+        /** 12.8 : 1 */
+        double angleGearRatio = (12.8 / 1.0);
+ 
+        double angleKP = 0.2;
+        double angleKI = 0.0;
+        double angleKD = 0.0;
+        double angleKF = 0.0;
+ 
+        boolean driveMotorInvert = false;
+        boolean angleMotorInvert = false;
+        boolean canCoderInvert = false;
+        return new COTSNeoSwerveConstants(wheelDiameter, angleGearRatio, driveGearRatio, angleKP, angleKI, angleKD, angleKF, driveMotorInvert, angleMotorInvert, canCoderInvert);
+    }
+
+    /** Swerve Drive Specialties - MK4i Module*/
+    public static COTSNeoSwerveConstants SDSMK4i(double driveGearRatio){
+        double wheelDiameter = Units.inchesToMeters(4.0);
+
+        /** (150 / 7) : 1 */
+        double angleGearRatio = ((150.0 / 7.0) / 1.0);
+
+        double angleKP = 0.3;
```

Comment:
Are these constants meant to be tweaked?  The Falcon version of this call had a P of 6.0 and a D of 0.15... Why is the Neo different here when the constants between Falcon and Neo are the same in the other calls?